weather.py

```diff
@@ -57,7 +68,20 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
+    weather_data = []
+    with open(csv_file) as this_file:
+        results = csv.reader(this_file)
+        count = 0
+        for line in results:
+            if line == []:
+                pass
+            elif count != 0:
```

Comment:
To improve code readability, this could also be an else statement. In this way, we are essentially only checking one condition (line == [] or not), rather than 2 (line == [] or not, and then count != 0 or not)